pkg/interactive/inputs/request_loader/model.go

```diff
@@ -0,0 +1,28 @@
+package requestloader
+
+import (
+	"context"
+	"time"
+
+	"github.com/charmbracelet/bubbles/spinner"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+	"github.com/apono-io/apono-cli/pkg/clientapi"
+)
+
+type model struct {
+	spinner        spinner.Model
+	ctx            context.Context
+	client         *aponoapi.AponoClient
+	request        *clientapi.AccessRequestClientModel
+	creationTime   time.Time
+	timeout        time.Duration
+	noWaitForGrant bool
+	quitting       bool
+	aborting       bool
+	err            error
+}
+
+type statusMsg clientapi.AccessRequestClientModel
```

Comment:
why you need this type alias? I think it's confusing and doesn't reflect the actual struct

Reply:
this is how the framework tell you to work with messages

Reply:
https://github.com/charmbracelet/bubbletea/tree/master/tutorials/commands/#commands-and-messages

Reply:
so maybe name it with `accessRequestModel` ?

Reply:
we decided on `updatedRequestMsg`